libs/sdk-ui-ext/styles/internal/icons/icon-pdf.svg

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Should we have icons in styles? Other svgs are in src/internal/assets. I encountered this too and put my icon in `incos` folder there: https://github.com/gooddata/gooddata-ui-sdk/pull/910/files#diff-21f4416b1b7b38d4d9ebd9ea548f6fbd40e95139ea0c0f9e6db830c014526caf

Reply:
Moved